### PR TITLE
Add Cloud SOAR Bridge article

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a new "Cloud SOAR Bridge" article:
https://help.sumologic.com/docs/cloud-soar/cloud-soar-bridge

Its content is almost identical to the Automation Service Bridge article:
https://help.sumologic.com/docs/cse/automation-service/automation-service-bridge/

At this point, no [content reuse](https://help.sumologic.com/docs/contributing/style-guide/#reusing-content) is done between the two articles. 

Issue number: 
[CSOAR-253](https://jira.kumoroku.com/jira/browse/CSOAR-253)
[TRIDENT-17883](https://jasklabs.atlassian.net/browse/TRIDENT-17883)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cloud-soar/cloud-soar-bridge.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+* OS: Ubuntu (18.04/20.04)
+* RAM: 8GB
+* CPU: 4 Core
+* DISK: 160GB
+* Network card: 1
+
+### Network requirements
+<!-- These network requirements differ from those for the Automation Service bridge -->
+
+The Bridge has to be able to resolve DNS hostnames and needs to reach the below destinations
```
Comment (kimsauce) [inline docs/cloud-soar/cloud-soar-bridge.md:26]:
```suggestion
The Bridge has to be able to resolve DNS hostnames and needs to reach the below destinations.
```

File: docs/cloud-soar/cloud-soar-bridge.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        "SOAR_TOKEN":"YOUR_JWT_TOKEN",
+        "SIEM_URL":"https://YOUR_CSE_URL/sec",
+        "ALIAS": "YOUR_ALIAS_NO_SPACES"
+}
+```
+Obtain the `SOAR_URL` by clicking **?** at the top of the Cloud SOAR UI and navigating to **API Documentation**. Note the **Servers** value and remove `/v3/` from the end of the URL. The bridge cannot currently be registered to a Cloud SOAR instance with the `/v3/` API. 
+
+### Bridge ALIAS
+
+With bridge ALIAS, it is possible to distinguish which integration resources will be executed with this automation bridge. When a new integration resource is created or edited it is possible to select the default ALIAS or to create a new one. So every automatic action configured to use this resource will be performed with the Bridge that has the same ALIAS.
```
Comment (kimsauce) [inline docs/cloud-soar/cloud-soar-bridge.md:115]:
```suggestion
With bridge ALIAS, it is possible to distinguish which integration resources will be executed with this automation bridge. When a new integration resource is created or edited, it is possible to select the default ALIAS or to create a new one. So every automatic action configured to use this resource will be performed with the Bridge that has the same ALIAS.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
looks great! minor suggestions.

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**